.eslintrc

```diff
@@ -0,0 +1,36 @@
+{
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
let's add https://www.npmjs.com/package/eslint-plugin-react here

Reply:
It was added, thanks